app/controllers/publishers/vacancies/base_controller.rb

```diff
@@ -21,7 +21,8 @@ def vacancies
   end
 
   def vacancy
-    @vacancy ||= vacancies.find(params[:job_id].presence || params[:id])
+    # Scope to internal vacancies to disallow editing of external ones
```

Comment:
🙋🏻  

Can I ask why we want to prevent editing external ones and who usually does it?

Reply:
Sure! The main reason in the short term is that we've made a decision to always consider the external system the source of truth, and every time the import runs it will overwrite the existing vacancy. This means that if someone were to edit the vacancy on our service, the next time the import runs their changes would be lost.